setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup, find_packages
+
+setup(name='http-request-randomizer',
+      version='0.0.3',
+      description='A package using public proxies to randomise http requests.',
+      url='https://pgaref.github.io/blog/python-proxy/',
+      author='pgaref',
+      author_email='pangaref@gmail.com',
+      license='MIT',
+      packages=find_packages(exclude=['tests']),
+      install_requires=["requests", "psutil", "httmock", "python-dateutil", "schedule"],
```

Comment:
Yes, but it is strange that it doesn't show up here. You can see it here: https://github.com/pgaref/HTTP_Request_Randomizer/pull/16/files